lib/utils/uid.js

```diff
@@ -4,8 +4,7 @@ let counter = 0
  * @link https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L16143
  * @param {string} prefix
  */
-export default function uid(prefix = "") {
+export default function uid(prefix = "bv_") {
     const id = ++counter
```

Comment:
The SSR stuff is a really good point. I would think an active node process would just keep incrementing the counter up to infinity (and beyond)!

I'm not sure if this will get used or not (and wiped), but I thought it might be handy to pluck this method off lodash for internal component keying in any lists. It's always nice to have one canonical way to do things. I was hearing some things about how vue tries to be efficient about rendering lists, and optimizes the process by reusing nodes. Long story short, I'm not sure if just using the index as a key really makes a node consistent (although it is unique). This is kind of pinned for more research later.

You may notice my dev style is to hoard my tools, get the job done, then trim everything down tight. If I end up implementing it anywhere, I'm gonna keep my eye open for SSR issues! That's a good point.